cmd/cmrel/cmd/stage.go

```diff
@@ -70,7 +70,7 @@ type stageOptions struct {
 	// The path to the cloudbuild.yaml file used to perform the cert-manager crossbuild
 	CloudBuildFile string
 
-	// Project to run the GCB job in
+	// GCP project to run the GCB job in
```

Comment:
suggestion: this breaks the doc comment although it improves the content of the comment. how about:

```suggestion
	// Project is the name of the GCP project to run the GCB job in
```

Reply:
Thanks for that! I've committed the change.